### ISSUE TITLE
New kind of Quiz

### ISSUE THREAD

**Issue by Mannshoch** [MazeChaZer/WoT-Tank-Quiz#1]
I saw your Quiz and got an Idea for an other one.

Since some time I started to Outsource Contacts and information from my Brain to my Mobil address book. But I realized more and more that this was not such a good Idea. In fact I have forgotten many about my contacts.

If you like it. Could you write a Quiz tool like you did for "World of Tank" for the Contact and Calendar Database?

**Easy Question:**
- Which Person is on this Photo?
- Which is the next Date for this Calendar entry?
- Choose the Photo of this Person
- Which Person(s) life in this town.

**Difficult Question:**
- On which date does this Person has his Birthday?
- Which Person life in this street.

**Realy Difficult Question**
- Which Phone Number belongs tho this Person?
- Enter the E-Mail address of this person

**Hardcor Questions**
- Enter the Phone Numer of this Person
  -  +49 ***

**Comment by MazeChaZer** [comment]
Hi, this sounds interesting, but unfortunately I haven't been working on this project for some years now. For the foreseeable future I won't be able to implement this, sorry.

**Comment by Mannshoch** [comment]
Yes I saw that you didn't worked for a long time but I hoped that the reason is only because you do not anymore an use for this app.

Well. I'm hoping and maybe if you have time, it would be nice and I'll do Betatesting if you need.  And If not, thats also OK.
